### DISCUSSION TITLE
How to launch a program with arguments in Nix?

### DISCUSSION THREAD

**Discussion by UnclePsymon** [xremap/xremap#517] [Q&A] [answered]
Hi! I want to launch programs with arguments. For example `kitty --hold ls`, but i can not get it to work. Does someone know how do do it? This is my config. Launching apps works perfektly fine without arguments.

```
keymap = [
        {
          name = "Apps";
          remap = {
            "super-alt-l" = {
              remap = {
                "super-alt-k" = {
                  launch = ["kitty"];
                };
                "super-alt-l" = {
                  launch = ["kitty --hold ls"];
                };
              };
            };
          };
        }
      ];
```

**Comment by onemoresuza** [comment, marked as answer]
Split the arguments to different elements of the list: `["kitty" "--hold" "ls"]` instead of `["kitty --hold ls"]`.
